### PR TITLE
NO-ISSUE: Add downstream verify makefile target

### PR DESCRIPTION
**Pull request by dtfranz**

### CODE REVIEW

File: openshift/Makefile [hunk 1]
```diff
@@ -7,6 +7,12 @@ include $(addprefix $(DIR)/vendor/github.com/openshift/build-machinery-go/make/,
 
 include $(DIR)/.bingo/Variables.mk
 
+.PHONY: verify
+verify: ## Run downstream-specific verify
+	$(MAKE) tidy fmt vet generate -C $(DIR)/../
+	$(MAKE) manifests
```
Comment (tmshort) [inline openshift/Makefile:13]: This could just be a dependency for `verify`, so no need for another make line... unless tidy, etc need to be run first.

Comment (joelanford) [reply openshift/Makefile:13]: They are run out of the upstream Makefile, so they aren't available here unless we `include` them (and including them might cause more trouble)

Comment (tmshort) [reply openshift/Makefile:13]: In this case, it's running the `verify` target in _this_ makefile (no `-C` option or `cd` command)

Comment (dtfranz) [reply openshift/Makefile:13]: That's right, it's running the downstream manifests target in this file. We do need to run tidy + fmt + vet + generate first though, so I think it needs to be kept in this order.

**Pull request merged**